cbot_actuators/launch/cbot_actuators.launch

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <launch>
     <rosparam file="$(find cbot_actuators)/config/params.yaml" command="load" />
-    <node name="thruster_node" pkg="cbot_actuators" type="thruster_node" output="screen">
-    </node>
+    <node name="thruster_node" pkg="cbot_actuators" type="thruster_node" respawn="true" output="screen"/>
+    <node name="thrusterSerial" pkg="cbot_actuators" type="thrusterSerial" respawn="true" output="screen"/>
```

Comment:
Snake case is preferred for file and variable names in C++, whereas camel case is used for class/function names